training/amd-bootc/Containerfile

```diff
@@ -2,7 +2,10 @@ FROM quay.io/centos-bootc/centos-bootc:stream9
 
 ADD rocm.repo /etc/yum.repos.d/rocm.repo
 
-RUN dnf install -y rocm-smi && dnf clean all
+ARG EXTRA_RPM_PACKAGES=''
+RUN dnf install -y rocm-smi \
```

Comment:
Doesn't
RUN dnf install -y rocm-smi ${EXTRA_RPM_PACKAGES}  && dnf clean all
Work?

Reply:
it certainly will :) Gonna update it